### PR TITLE
MapBox CSS dark theme compatibility and JS syntax error fix

### PR DESCRIPTION
**Pull request by cebence**
Eclipse JavaScript compiler / editor is reporting a syntax error in control's JS file. Fixed.

Explicitly setting control's background color breaks compatibility with easily switching to MapBox dark theme. Fixed by setting / overriding only background image properties.

### CODE REVIEW

**Comment by tmcw** [comment]
Can you split these changes? The JavaScript fix is unnecessary and `{ true: 'value' }` is valid, but the CSS fix is correct and useful.

**Comment by cebence** [comment]
Regarding the JS fix, I only copied the JS compressor's output to the source so there are no syntax errors.
If you really don't need the JS fix I can probably split them, but I haven't done it before so any advice is appreciated.

**Comment by tmcw** [comment]
Yes, to be clear, Eclipse's JavaScript parser/compressor is incorrect and this isn't a fix in JavaScript as much as a bug in its implementation.

**Comment by cebence** [comment]
I can't split this via Github web interface, I need to do it localy using git bash, but I am unable to do that due to problems with my company proxy's self-signed certificate.

Also, I checked [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object), `title` is an object with two properties named "true" and "false". The compressor that generated `/dist/Leaflet.fullscreen.min.js` followed the rules and quoted them - it's the tools that are lenient in allowing unquoted property names.

A small example:

```
var title = {
  false: 'View fullscreen',
  true: 'Exit fullscreen'
};
for (var i in title) {
  console.log(i + ':' + (typeof i) + ' = ' + title[i]);
}
```

And the output (Google Chrome) is:

```
false:string = View fullscreen
true:string = Exit fullscreen
```

Given this, I really don't see an issue here.

**Comment by tmcw** [comment]
Please see the [JavaScript standard if you want to learn about this in detail](http://www.ecma-international.org/ecma-262/5.1/#sec-11.1.5): the initialization syntax of an object is not equal to the produced object. I'll port the CSS changes.

**Comment by cebence** [comment]
[Object Initializer example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects#Using_object_initializers) containing quoted property names - JSLint accepts it both ways.

**Most importantly**, Eclipse / Aptana is actually pointing the use of a reserved word as property name - boolean literals, i.e. `true` and `false`, are reserved words [per ECMA 5.5, section 7.6.1](http://www.ecma-international.org/publications/files/ECMA-ST-ARCH/ECMA-262%205th%20edition%20December%202009.pdf) or [summary](http://mathiasbynens.be/notes/reserved-keywords).

In summary, the code performs identically, it's just the red error mark that is going away.
Thank you.
